src/components/form/ComboBox/ComboBox.tsx

```diff
@@ -73,6 +74,13 @@ export const ComboBox = ({
     matchFrom: allowAddValue ? 'start' : 'any',
     stringify: (option) => option.label || option.value,
   })
+  const startAdornmentValue = useMemo(() => {
+    let foundGroup = data.find((item) => item.value === value)?.group
```

Comment:
Sorry 😅

But to avoid unnecessary calculation, can we :
```
if (!renderGroupInputStartAdornment || !value) return undefined 
```

before any check ?

Reply:
sure thing !

Reply:
done